templates/tfengine/components/devops/main.tf

```diff
@@ -125,7 +131,11 @@ module "admins_group" {
   display_name = "{{regexReplaceAll "@.*" .admins_group.id ""}}"
   {{- end}}
   {{hclField .admins_group "description" -}}
-  {{hclField .admins_group "owners" -}}
+  {{if has .admins_group "owners" -}}
```

Comment:
Can we simply not create the owners group and just leave the default owner on the project (which will be the caller)?

Reply:
I think it's better to always create a placeholder group during initial devops deployment and assign owner role to it so future owner permission granting does not necessarily go though IAM but through adding to the group.

And for org/folder admin role, we have to explicitly grant the IAM role to the group/individual anyway. So it's more consistent to do the same for devops owner role.